test/e2e/specs/specs-playwright/wp-signup__screenshot.ts

```diff
@@ -0,0 +1,299 @@
+/**
+ * @group calypso-release
```

Comment:
This looks like a utility script and so I would like to propose that this be moved out of the pre-release tests group and instead put into a new category, such as `compliance` or `legal`.

I can help set up a new TeamCity task for this once the script is done and ready.

Reply:
I've changed this to a new "legal" group.